index.html

```diff
@@ -963,12 +963,12 @@ <h2>DID Parameters</h2>
 
         <pre class="example nohighlight"
           title="A DID URL with a 'versionTime' DID parameter">
-did:example:123?versionTime=2021-05-10T17:00:00Z
+did:example:123?versionTime=2021-05-10T17%3A00%3A00Z
```

Comment:
don't need to encode colons in the query segment
```suggestion
did:example:123?versionTime=2021-05-10T17:00:00Z
```